server/server/src/lib.rs

```diff
@@ -119,7 +119,7 @@ pub async fn start_server(
             .unwrap();
     }
 
-    logging_init(settings.logging.clone(), None, log_level);
+    logging_init(settings.logging.clone(), log_level, true);
```

Comment:
Any reason why this one is config update ?

Reply:
An endpoint to update log level was added [here](https://github.com/openmsupply/open-msupply/issues/1890) and the UI changes will be implemented [here](https://github.com/openmsupply/open-msupply/issues/1891). Will add comment...

Though I was thinking... We could save the log settings into `key_value_store` and have the log level updated when updateLogLevel is called... that way we don't need to tell the user to restart the server for log changes... What do you think? 🤔